compatibility with new python-docker

```
=================================== FAILURES ===================================
________________________________ test_short_id _________________________________

    def test_short_id():
        mock()
>       b = DockerBackend()

test_container_info.py:8: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <sen.docker_backend.DockerBackend object at 0x7f6c7cbac050>

    def __init__(self):
        self._containers = None
        self._images = None  # displayed images
        self._all_images = None  # docker images -a
        self._df = None
    
        kwargs = {"version": "auto"}
>       kwargs.update(docker.utils.kwargs_from_env(assert_hostname=False))
E       TypeError: kwargs_from_env() got an unexpected keyword argument 'assert_hostname'

../sen/docker_backend.py:794: TypeError
```

```
[7/9] Installing python3-docker-0:7.1.0 100% |  16.3 MiB/s |   1.1 MiB |  00m00s
```

--- a/sen/docker_backend.py
+++ b/sen/docker_backend.py
@@ -791,7 +791,7 @@ class DockerBackend:
         self._df = None
 
         kwargs = {"version": "auto"}
-        kwargs.update(docker.utils.kwargs_from_env(assert_hostname=False))
+        kwargs.update(docker.utils.kwargs_from_env())
 
         try:
             APIClientClass = docker.Client  # 1.x

--- a/tests/real.py
+++ b/tests/real.py
@@ -29,16 +29,38 @@ container_data = {
     'Status': 'Exited (0) 47 hours ago'
 }
 
-
 version_data = {
-    'ApiVersion': '1.21',
-    'Arch': 'amd64',
-    'BuildTime': 'Thu Sep 10 17:53:19 UTC 2015',
-    'GitCommit': 'af9b534-dirty',
-    'GoVersion': 'go1.5.1',
-    'KernelVersion': '4.2.5-300.fc23.x86_64',
-    'Os': 'linux',
-    'Version': '1.9.0-dev-fc24'
+  'Platform': {'Name': 'linux/amd64/fedora-40'},
+  'Components': [{
+    'Name': 'Podman Engine',
+    'Version': '5.1.2',
+    'Details': {
+      'APIVersion': '5.1.2',
+      'Arch': 'amd64',
+      'BuildTime': '2024-07-10T02:00:00+02:00',
+      'Experimental': 'false',
+      'GitCommit': '',
+      'GoVersion': 'go1.22.5',
+      'KernelVersion': '6.9.11-200.fc40.x86_64',
+      'MinAPIVersion': '4.0.0',
+      'Os': 'linux'}},
+    {
+      'Name': 'Conmon',
+      'Version': 'conmon version 2.1.10, commit: ',
+      'Details': {'Package': 'conmon-2.1.10-1.fc40.x86_64'}},
+    {
+      'Name': 'OCI Runtime (crun)',
+      'Version': 'crun version 1.15\ncommit: e6eacaf4034e84185fd8780ac9262bbf57082278\nrundir: /run/user/1000/crun\nspec: 1.0.0\n+SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +LIBKRUN +WASM:wasmedge +YAJL',
+      'Details': {'Package': 'crun-1.15-1.fc40.x86_64'}}],
+  'Version': '5.1.2',
+  'ApiVersion': '1.41',
+  'MinAPIVersion': '1.24',
+  'GitCommit': '',
+  'GoVersion': 'go1.22.5',
+  'Os': 'linux',
+  'Arch': 'amd64',
+  'KernelVersion': '6.9.11-200.fc40.x86_64',
+  'BuildTime': '2024-07-10T02:00:00+02:00'
 }
 
 top_data = {

